testsuite/Main.hs

```diff
@@ -50,6 +50,7 @@ main =
             , runTest ("tests" </> "shouldwork" </> "Basic") defBuild [] "ClassOps"            (["","ClassOps_testBench"],"ClassOps_testBench",True)
             , runTest ("tests" </> "shouldwork" </> "Basic") defBuild [] "CountTrailingZeros"  (["","CountTrailingZeros_testBench"],"CountTrailingZeros_testBench",True)
             , runTest ("tests" </> "shouldwork" </> "Basic") defBuild [] "DivMod"              ([""],"DivMod_topEntity",False)
+            , runTest ("tests" </> "shouldwork" </> "Basic") defBuild [] "FromJust"            ([""],"FromJust_topEntity",False)
```

Comment:
Change this line to:

```haskell
, runTest ("tests" </> "shouldwork" </> "Basic") defBuild [] "FromJust"            (["FromJust_topEntity"],"FromJust_topEntity",False)
```

or, remove the `TopEntity` annotation in your test, and rename `dut` to `topEntity`.